libr/core/panels.c

```diff
@@ -106,7 +106,29 @@ static const char **menus_sub[] = {
 	NULL
 };
 
-static void Panel_print(RConsCanvas *can, RPanel *panel, int color) {
+static void layoutRun(RPanels *panels);
+static void panelPrint(RConsCanvas *can, RPanel *panel, int color);
+static void addPanelFrame(RPanels* panels, const char *title, const char *cmd);
+static void check_stackbase(RCore *core);
+static void cursor_left(RCore *core);
+static void cursor_right(RCore *core);
+static void delcurpanel(RPanels *panels);
+static void doRefresh(RPanels *panels);
+static bool handle_cursormode(RCore *core, const int key);
```

Comment:
all static functions should be camelcase imho

Reply:
agreed and fixed like that